File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/ServerLease.scala

package org.openapitools.client.model


case class ServerLease (
    /* MAC address associated with the lease. */
    _mac: String,
    /* Indicates if the lease is authenticated. */
    _authenticated: Boolean,
    /* Group identifier for the lease. */
    _group: String
)
object ServerLease {
    def toStringBody(var_mac: Object, var_authenticated: Object, var_group: Object) =
        s"""
        | {
        | "mac":$var_mac,"authenticated":$var_authenticated,"group":$var_group
        | }
        """.stripMargin
}

File: OPENAPI-REST-API/openapi-client/zapier/models/ServerLease.js
const utils = require('../utils/utils');

module.exports = {
    fields: (prefix = '', isInput = true, isArrayChild = false) => {
        const {keyPrefix, labelPrefix} = utils.buildKeyAndLabel(prefix, isInput, isArrayChild)
        return [
            {
                key: `${keyPrefix}mac`,
                label: `MAC address associated with the lease. - [${labelPrefix}mac]`,
                required: true,
                type: 'string',
            },
            {
                key: `${keyPrefix}authenticated`,
                label: `Indicates if the lease is authenticated. - [${labelPrefix}authenticated]`,
                required: true,
                type: 'boolean',
            },
            {
                key: `${keyPrefix}group`,
                label: `Group identifier for the lease. - [${labelPrefix}group]`,
                required: true,
                type: 'string',
            },
        ]
    },
    mapping: (bundle, prefix = '') => {
        const {keyPrefix} = utils.buildKeyAndLabel(prefix)
        return {
            'mac': bundle.inputData?.[`${keyPrefix}mac`],
            'authenticated': bundle.inputData?.[`${keyPrefix}authenticated`],
            'group': bundle.inputData?.[`${keyPrefix}group`],
        }
    },
}
